mashed-potatoes.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <a href="index.html">HOME</a>
+    <h1>MASHED POTATOES RECIPE</h1>
+    <img
+      src="images/MASHED POATOES.jpeg"
+      alt="a picture of creamy mashed potatoes"
+    />
+    <h2>INGREDIENTS:</h2>
+    <ul>
+      <li>1.5 kg / 3 lb potatoes, peeled and cut into 2.5cm/1” cubes</li>
+      <li>1 tbsp salt</li>
+      <li>60g / 4 tbsp unsalted butter</li>
+      <li>1/2 cup milk</li>
+      <li>Chives or parsley</li>
+    </ul>
+    <body>
+      <h3>PREPARATION TIME</h3>
+      <ul>
+        <li>Prep: 10minutes mins</li>
+        <li>Cook: 20minutes mins</li>
+      </ul>
+      <body>
+        <h4>INSTRUCTIONS</h4>
+        <ul>
+          <li>
+            Place in a large pot with 1 tbsp salt. Add water so it’s 10cm / 4”
+            above potatoes.
+          </li>
+          <li>
+            Bring to a boil over high heat then reduce heat so it’s simmering
+            rapidly. Cook 15 minutes or until potatoes are very soft (jab with
+            fork to test, they should fall apart).
+          </li>
+          <li>
+            Drain well, return into pot. Leave for 1 minute, shaking pot every
+            now and then, to encourage evaporation of water.
+          </li>
+          <li>
+            Add Flavourings then mash well, using milk to make it looser if
+            desired.
+          </li>
+        </ul>
+      </body>
+    </body>
+  </body>
```

Comment:
this is highly incorrect. It possibly leads to incorrectly displaying your pages in any given browser. I would avoid this to avoid irratic behaviour - especially since multiple body tags don't serve any semantic use-case as well